ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -216,23 +217,143 @@ public int getMaxTrailingPeers() {
   public static class Builder {
     private SyncMode syncMode = SyncMode.FULL;
     private Range<Long> blockPropagationRange = Range.closed(-10L, 30L);
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-downloader-change-target-threshold-by-height",
+        hidden = true,
+        defaultValue = "20",
```

Comment:
I take it we wound up having to have the defaults duplicated then?  Should we raise an issue for PicoCLI so hopefully we can have them magically picked up at some point in the future?

Reply:
I didn't look too deeply.  There is support for a defaults provider but getting it linked to the fields that were annotated is a good bit of work.